src/common/localization.js

```diff
@@ -464,6 +481,15 @@ const STRINGS = new LocalizedStrings({
     DOWNLOAD: "ਡਾਊਨਲੋਡ",
     DOWNLOADING: "ਡਾਊਨਲੋਡ ਹੋ ਰਹਿ ਰਹਾ ਹੈ",
     NEXT: "ਅਗਲਾ",
+    PLAYBACK_SPEED: "ਪਲੇਬੈਕ ਸਪੀਡ",
+    UNABLE_TO_PLAY: "ਆਡੀਓ ਚਲਾਉਣ ਲਈ ਅਸਮਰਥ",
+    PLEASE_TRY_AGAIN: "ਕੀਪਾ ਪੁਨਰਪਾਤ ਕਰੋ।",
+    UNABLE_TO_SEEK: "ਆਡੀਓ ਖੋਜਣ ਲਈ ਅਸਮਰਥ",
+    UNABLE_TO_SWITCH_TRACK: "ਆਡੀਓ ਟਰੈਕ ਬਦਲਣ ਲਈ ਅਸਮਰਥ",
+    AUDIO_SYNC_UNAVAILABLE: "ਆਡੀਓ ਸਿੰਕ ਅਸਵੀਕਰਿਤ ਹੈ। ਆਡੀਓ ਑ਟੋ ਸਕਰੋਲ ਦੇ ਬਿਨਾਂ ਚਲਾਈ ਜਾਵੇਗੀ।",
+    NETWORK_ERROR:
+      "ਨੈੱਟਵਰਕ ਤਰੁੱਟੀ। ਆਡੀਓ ਸੰਸਕਰਨਾਂ ਨੂੰ ਅਸਥਾਈ ਰੂਪ ਸਕਰੋਲ ਦੇ ਬਿਨਾਂ ਅਸਵੀਕਰਿਤ ਕਰ ਦਿੱਤਾ ਜਾਵੇਗਾ।",
+    COULD_NOT_LOAD_AUDIO_ARTISTS: "ਆਡੀਓ ਕਲਾਰਾਰਾਂ ਨੂੰ ਲੋਡ ਕਰਨ ਲਈ ਅਸਮਰਥ। ਕੀਪਾ ਆਪਣਾ ਕਨੈਕਸ਼ਨ ਜਾਂਚੋ।",
```

Comment:
Spelling of ਕੀਪਾ is wrong here too. It might help to do a quick run through all the translations and check the wording/translations.

Reply:
fixed